react/features/connection-indicator/components/web/ConnectionIndicatorContent.tsx

```diff
@@ -26,6 +27,8 @@ import AbstractConnectionIndicator, {
     INDICATOR_DISPLAY_THRESHOLD
 } from '../AbstractConnectionIndicator';
 
+import BandwidthSettingsDialog from './BandwidthSettingsDialog';
```

Comment:
Is the connection indicator where we settled on displaying this? Wouldn't the performance dialog be more appropriate?

Reply:
Yes, we wanted it to be a little bit more difficult to stumble upon.